modules/dns_nsupdate/dns_nsupdate_plugin.rb

```diff
@@ -0,0 +1,13 @@
+module ::Proxy::Dns::Nsupdate
+  class Plugin < ::Proxy::Plugin
+    plugin :nsupdate, ::Proxy::VERSION
+    default_settings :enabled => true, :dns_server => 'localhost', :dns_ttl => 86400
+
+    requires :dns, '~> 1.8.0'
```

Comment:
Maybe for core providers we should just use ::Proxy::VERSION so it's a 1:1 match.